src/equations/compressible_navier_stokes_3d.jl

```diff
@@ -0,0 +1,380 @@
+@doc raw"""
+    CompressibleNavierStokesDiffusion3D(gamma, mu, Pr, equations,
```

Comment:
`@muladd` stuff?

Reply:
We also do not have the `@muladd`  in the 2D Compressible Navier-Stokes equations. I cannot recall why we didn't put that in.

Reply:
I think we just hadn't added it yet. I suggest we merge this first since the PR is getting large and open an issue to add `@muladd`

Reply:
I made a new issue #1277 that references this.